### PR TITLE
Converted puppet forge to use Django web framework

### PR DESCRIPTION
**Pull request by dkliban**
Closes #96

### CODE REVIEW

**Comment by dkliban** [comment]
ok test

**Comment by dkliban** [comment]
ok test

File: pulp_puppet_plugins/pulp_puppet/forge/middleware/requesturi.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Copyright © 2011 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
```
Comment (barnabycourt) [inline pulp_puppet_plugins/pulp_puppet/forge/middleware/requesturi.py]: copyright blob should not be included

**Comment by barnabycourt** [comment]
pulp-dev.py should be updated for these changes.
pulp-puppet.spec should be updated for these changes.
The old wsgi files should be removed if they are no longer used

**Comment by barnabycourt** [comment]
Why the duplication between pulp_puppet_plugins/pulp_puppet/forge/wsgi.py and pulp_puppet_plugins/srv/pulp/puppet_forge.wsgi?

**Comment by barnabycourt** [comment]
I'd flake8 your updates and then call it good

**Pull request merged**